client/src/Booking.jsx

```diff
@@ -29,23 +31,24 @@ class Booking extends React.Component {
       weeknightDiscount: 0,
       year: 0
     };
-    this.fetcher = this.fetcher.bind(this);
+    this.init= this.init.bind(this);
+    this.book = this.book.bind(this);
   }
 
   componentDidMount() {
-    this.fetcher();
+    this.init();
+    this.book();
   }
 
-  fetcher() {
-    axios.get('/booking', { params: { campId: 0 } })
+  init() {
+    return axios.get('http://localhost:3002/booking', { params: { campId: 0 } })
```

Comment:
yay my error went away but now i'm getting type is not defined

Reply:
rad now that this is will be merged I can fix the type error TY